includes/class-wc-calypso-bridge-setup.php

```diff
@@ -35,6 +35,12 @@ public static function get_instance() {
 	 * Constructor.
 	 */
 	private function __construct() {
+		// If setup has yet to complete, make sure MailChimp doesn't redirect the flow.
+		$has_finshed_setup = (bool) get_option( 'atomic-ecommerce-setup-checklist-complete' );
```

Comment:
Can we use `is_checklist_done()` from the class `WC_Calypso_Bridge_Admin_Setup_Checklist`?  I've introduced some new logic in https://github.com/Automattic/wc-calypso-bridge/pull/479 that would change what qualifies as complete.

We can probably change it to a static method.